pos_report_session_summary/views/report_session_summary.xml

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+<template id="report_session_summary">
+    <t t-call="web.html_container">
+        <t t-foreach="docs" t-as="o">
+            <t t-call="web.external_layout">
+                <div class="page">
+                    <h2>Session Summary: <span t-field="o.name" /></h2>
+
+                    <div class="row mt32 mb32">
+                        <div class="col-auto mw-100 mb-3">
+                            <strong>Responsible</strong>:<br />
+                            <span t-field="o.user_id" />
+                        </div>
+                        <div class="col-auto mw-100 mb-3">
+                            <strong>Point of Sale</strong>:<br />
+                            <span t-field="o.config_id" />
+                        </div>
+                        <div class="col-auto mw-100 mb-3">
+                            <strong>Opening Date</strong>:<br />
+                            <span t-field="o.start_at" />
+                        </div>
+                        <div class="col-auto mw-100 mb-3">
+                            <strong>Closing Date</strong>:<br />
+                            <span t-field="o.stop_at" />
+                        </div>
+                    </div>
+
+                    <h4>Statement Summary</h4>
+
+                    <table class="table table-sm o_main_table">
+                        <thead>
+                            <tr>
+                                <th>
+                                    <strong>Reference</strong>
+                                </th>
+                                <th>
+                                    <strong>Journal</strong>
+                                </th>
+                                <th class="text-end">
+                                    <strong>Starting Balance</strong>
+                                </th>
+                                <th class="text-end">
+                                    <strong>Total Transactions</strong>
+                                </th>
+                                <th class="text-end">
+                                    <strong>Ending Balance</strong>
+                                </th>
+                                <th class="text-end">
+                                    <strong>Difference</strong>
+                                </th>
+                                <th class="text-end" groups="base.group_multi_currency">
+                                    <strong>Currency</strong>
+                                </th>
+                                <th class="text-end">
+                                    <strong>Status</strong>
+                                </th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            <tr
+                                    t-foreach="o.statement_line_ids.mapped('statement_id')"
```

Comment:
I think it is better to use `pos.payment` instead of `account.bank.statement.line`. Please review this [model and field](https://github.com/odoo/odoo/blob/62d75df1495b85f75522f7c077e1b3a93fe800af/addons/point_of_sale/models/pos_payment.py#L26
). If `pos.session` does not have a field to `pos.payment`, you can add this field as `one2many` or include it in the report by performing a `read_group` to get all related payment information.
Remember that `account.bank.statement` is no longer required and Odoo does not create records in this model from POS.
![image](https://github.com/user-attachments/assets/a177f0ef-5ec4-441d-aa85-d700d0cdc74f)